Dockerfile

```diff
@@ -28,7 +43,7 @@ RUN addgroup --gid 2000 --system appgroup && \
 WORKDIR /app
 COPY --from=builder --chown=appuser:appgroup /app/build/libs/hmpps-interventions-service*.jar /app/app.jar
 COPY --from=builder --chown=appuser:appgroup /app/build/libs/applicationinsights-agent*.jar /app/agent.jar
-COPY --from=builder --chown=appuser:appgroup /app/applicationinsights.json /app
+COPY --chown=appuser:appgroup applicationinsights.json /app
```

Comment:
It's OK to copy this file from the filesystem directly, and perhaps cleaner as well